src/components/Dropdown/DropdownList.js

```diff
@@ -83,7 +83,7 @@ class DropdownList extends React.PureComponent {
     return this.hoverCallbacks[itemKey];
   };
 
-  handleSelectKeyUse = () => {
+  handleSelectKeyUse = event => {
```

Comment:
maybe we should keep convention and decide to use `e` or `event`?

Reply:
yup, good idea. Let's use `event` :)

Reply:
changed in https://github.com/livechat/design-system/pull/80/commits/056f8e8d2dba917e061ded9e4c6bdabc2db8bc0c